src/app/interfaces/dynamic-form-schema.interface.ts

```diff
@@ -33,13 +33,15 @@ export interface DynamicFormSchemaInput extends DynamicFormSchemaBase {
   type: DynamicFormSchemaType.Input;
   tooltip?: string;
   private?: boolean;
+  number?: boolean;
```

Comment:
Maybe we ought to have something like `type` instead of `private` and `number` with enum of values like `password` and `number`. Would be more inline with how ix-input works.